src/config/navigationOptions.js

```diff
@@ -18,6 +26,17 @@ export const bookDetailNavigationOptions = {
 };
 
 export const bookListNavigationOptions = {
-    title: LIBRARY
+    title: LIBRARY,
+    tabBarLabel: LIBRARY,
+    tabBarIcon: ({ focused }) => <Image style={{ width: SPACING_MEDIUM, height: SPACING_MEDIUM }} source={focused ? libraryActiveImage : libraryImage } />
```

Comment:
Extract `TabBarIcon` into an app component. You can pass via props the activeIcon and normalIcon